docs(graphql): subscriptions server side filtering and multiple owners support (#4696)
aws-amplify/docs: src/fragments/lib/graphqlapi/js/subscribe-data.mdx

});
```

> Subscriptions take an optional `filter` argument which provides an option to define and enable business logic for data filtering on the backend directly.

Reviewer: ```suggestion
Subscriptions take an optional `filter` argument to define service-side subscription filters:
```

```graphql

> // Subscribe to creation of Todo

Reviewer: ```suggestion
// Only subscribe to creation of Todo, if the todo "type" is "Personal"
```

const subscription = API.graphql(
    graphqlOperation(subscriptions.onCreateTodo, {
      filter: {
        type: { eq: "Personal" }
      }
    })
).subscribe({
    next: ({ provider, value }) => console.log({ provider, value }),
    error: error => console.warn(error)
});
```

> Generally, don’t pass the `filter` parameter if you want to get all subscription events.

Reviewer: ```suggestion
If you want to get all subscription events, don’t pass any `filter` parameters.
```

<Callout>

> **Important**: Passing an empty object `{}` is NOT recommended because it means “no filter” only when multiple owner authorization is used.

Reviewer: ```suggestion
**Important**: Passing an empty object `{}` as a filter is NOT recommended because it means “no filter” only when multiple owner authorization is used.
```
